pageserver/src/tenant/mgr.rs

```diff
@@ -446,7 +489,15 @@ pub(crate) fn schedule_local_tenant_processing(
                 "attaching mark file present but no remote storage configured".to_string(),
             )
         } else {
-            match Tenant::spawn_attach(conf, tenant_id, resources, location_conf, tenants, ctx) {
+            match Tenant::spawn_attach(
+                conf,
+                tenant_id,
+                resources,
+                location_conf,
+                tenants,
+                ctx,
+                true,
+            ) {
```

Comment:
Please use an enum instead of naked bools, it makes the call site more readable.

Also, please leave ctx as the last argument at all times, it's the convention we (mostly) follow in the code base. Should probably write that down somewhere, but where?

Reply:
45067d9d2f1a9afb0ed9b5fe012b31435c26b7c0